build-settings/rollup.config.js

```diff
@@ -267,7 +267,9 @@ const getPkgNames = (commandLineArgs) => {
     const {configPackages} = commandLineArgs;
 
     // Get the list of packages that we have in our packages folder.
-    const actualPackages = fs.readdirSync("packages");
+    const actualPackages = fs
+        .readdirSync("packages")
+        .filter((name) => name.startsWith("wonder-stuff-"));
```

Comment:
There's no build step for `eslint-config-khan`.  I'll add a comment in the code about this.

Reply:
It doesn't feel like this is how we want to configure this, as otherwise this is going to fail (in a very confusing way) the next time we add a non-wonder-stuff package that does have a build step. Instead, it would be better to add something to the `package.json` indicating that there is no build step, which we then read in `getPackageInfo` and filter out before we call `createConfig` in `createRollupConfig`.

Reply:
Good idea.